extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -161,6 +169,15 @@ export class AmpLiveList extends AMP.BaseElement {
     /** @private {?Element} */
     this.pendingPagination_ = null;
 
+    /**
+     * Element that dynamically built this amp-live-list (if any).
+     * @private {?Element}
+     */
+    this.listBuilder_ = null;
```

Comment:
nit: the "builder" suffix sounds to me like a utility class that is going to help build the list (maybe it's my Java side coming out...)

Can we at least add an `el` or `element` suffix so it's clear that this is an element?

Reply:
Done.